Surface_mesh_simplification/doc/Surface_mesh_simplification/Concepts/Filter.h

```diff
@@ -0,0 +1,41 @@
+/*!
+\ingroup PkgSurfaceMeshSimplificationConcepts
+\cgalConcept
+
+The concept `Filter` describes the requirements for the <I>policy
```

Comment:
I wonder if the concept shouldn't be named `PlacementFilter` as `Filter` is exposed at the global scope of concepts.

Reply:
I just  changed this.